features/Other/Edit_Info_Person_Affected_From_EspoCRM.feature

```diff
@@ -0,0 +1,35 @@
+@swagger-ui
+Feature: Edit info From ESPO CRM
+
+Background:
+Given a user in the Swagger UI
+
+Scenario: Successfully Update FSP from Whatsapp to Jumbo from EspoCRM
+
+    Given the "programId" is provided
+    Given the "refferenceId" is provided
+    Given the "NewFspAttributes" is provided
+    When 'newFspName' value is entered as 'Intersolve-jumbo-physical'
+    And the user has filled in values for "NewFspAttributes"
```

Comment:
I believe the idea here is that there can only 1 'When' statement (and thus that there cannot be 'And's after a 'When'), as the 'When' should really pinpoint 1 specific action that is done by the user in a scenario. So all 'preparation' before this action should be under 'Given'. (Same for the other scenarios below.)